docs/Changelog.md

```diff
@@ -27,6 +27,8 @@ sending a mail to <afl-users+subscribe@googlegroups.com>.
     - add AFL_QEMU_CUSTOM_BIN
   - unicorn_mode
     - accidently removed the subfolder from github, re-added
+  - added DEFAULT_PERMISSION to config.h for all files created, default
```

Comment:
`DEFAULT_PERMISSION` with backticks? Same for `0600`

Reply:
that would be then needed for most lines in the changelog ... nah :) rather spend my time otherwise :) but feel free!